docs/REST-API/99-Rate-Limiting-23.md

```diff
@@ -0,0 +1,74 @@
+---
+tags: [rest-api]
+---
+
+# REST API Rate Limits
+
+<!-- theme: warning -->
+> ### Early Access
+>
+> This page contains the updates that will be published to our [Rate Limiting](../../docs/REST-API/04-Rate-Limiting.md) documentation page as we start 
+> to roll out the 2023 Rate Limiting upgrades.
+
+## What are the limits?
+
+The PagerDuty REST API uses rate limits in order to provide a consistent experience for all of our lovely customers. The limits depend on the type of authorization token being used. Each PagerDuty App has its own set of limits that are separate from both API keys and other apps.
```

Comment:
please remove "lovely".

Reply:
But Rate Limiting is supposed to be fun 😜 .